document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -357,17 +362,16 @@ public CloseableIterator<Document> search(Query query) {
     }
 
     String pgSqlQuery = sqlBuilder.toString();
-    try {
-      PreparedStatement preparedStatement =
-          buildPreparedStatement(pgSqlQuery, paramsBuilder.build());
-      LOGGER.debug("Executing search query to PostgresSQL:{}", preparedStatement.toString());
```

Comment:
Can we keep this debug log? May with pgSqlQuery and paramBuilder?

Reply:
Dne